client/coral-framework/actions/embedStream.js

```diff
@@ -0,0 +1,22 @@
+import {base, handleResp, getInit} from '../helpers/response';
```

Comment:
I feel like that architecture got us into trouble with Ask. We had a "Forms" action/reducer that went crazy as we added features. I advocate breaking things up by how they're stored in the db like `Users` `Comments` or whatever instead of matching up to a view. Also if we want to move some actions to another file, it makes it weird.